spaceship/lib/spaceship/test_flight/group.rb

```diff
@@ -37,11 +37,11 @@ def self.filter_groups(app_id: nil, &block)
     end
 
     def add_tester!(tester)
-      client.add_tester_to_group!(group: self, tester: tester, app_id: self.app_id)
+      client.put_test_to_group(group_id: self.id, tester_id: tester.tester_id, app_id: self.app_id)
     end
 
     def remove_tester!(tester)
-      client.remove_tester_from_group!(group: self, tester: tester, app_id: self.app_id)
+      client.delete_tester_from_group(group_id: self.id, tester_id: tester.tester_id, app_id: self.app_id)
```

Comment:
Yeah, the methods were renamed in client.rb, and we haven't updated the parts that call it...